app/helpers/discussions_helper.rb

```diff
@@ -122,7 +122,9 @@ def discussions_statuses
   #TODO: this one uses a long method chain in order to take advantage of eager load
   # Delegate it once again when polymorphic association is removed
```

Comment:
Is this still relevant?

Reply:
We _do_ still have a polymorphic association in discussions. We're only ever using it with exercises though; Not sure what they meant by delegating it again, but we _could_ remove the item_type from discussions if we don't plan on ever associating them to anything else